src/styles/components/dropdown.scss

```diff
@@ -8,6 +8,7 @@
 	--text-dropdown-button: var(--text-color);
 	--text-dropdown-button-color: var(--dark-black);
 	--toggle-text-color: var(--off-white);
+	--primary-txt: var(--white);
```

Comment:
@aringocode Rename it to `--reversed-text-dropdown-color` to stick with the naming convention